### PR TITLE
{math}[intel/2017b] graph-tool v2.26 (WIP)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
WIP since there are a couple of compilation errors I haven't figured out yet...

/me eyes @wpoely86 ;)

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2459.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/71670a95826cc096d0fb00dbcd558263 for a full test report.

**Comment by wpoely86** [comment]
```
checking for Expat XML Parser... no
configure: error: expat library not found
```

missing dep?

File: easybuild/easyconfigs/g/graph-tool/graph-tool-2.26-intel-2017b-Python-3.6.3.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+dependencies = [
+    ('Python', '3.6.3'),
+    ('Boost', '1.65.1', versionsuffix),
+    ('expat', '2.2.4'),
+    ('CGAL', '4.11', versionsuffix),
+    ('sparsehash', '2.0.3'),
+]
+
+configopts = '--enable-openmp --disable-cairo --with-boost=$EBROOTBOOST '
```
Comment (wpoely86) [inline easybuild/easyconfigs/g/graph-tool/graph-tool-2.26-intel-2017b-Python-3.6.3.eb]: Needs a `--with-expat=$EBROOTEXPAT`?

Comment (boegel) [reply easybuild/easyconfigs/g/graph-tool/graph-tool-2.26-intel-2017b-Python-3.6.3.eb]: Fixed, but still same issues on my end, please try again?

**Comment by boegel** [comment]
closing this in favour of #5779 using `foss/2017b`

**Pull request closed without merging**